web/src/services/Access.ts

```diff
@@ -48,7 +48,7 @@ const PermissionSideMenuItemMap: Record<string, SideMenuItem[]> = {
     ],
     "FullAccess": [
         {
-            label: "Apontaments ADM",
+            label: "Apontamentos Geral",
             url:   'http://localhost:7000/Appointments' }
```

Comment:
remover base da url pois será definida através de variaveis de ambiente
```suggestion
            url:   '/Appointments' }
```